### DISCUSSION TITLE
Error when playing files using a linked directory

### DISCUSSION THREAD

**Discussion by MP3GMike** [ant-media/Ant-Media-Server#5056] [Q&A] [answered]
Hi I have the following problem:

I'm using the linked directory and is not working with files with this kind of name:

![image](https://user-images.githubusercontent.com/19702770/235527667-3c2d9bd5-c94c-4c13-abda-481099808261.png)

when I try to play those files it never starts and gives an error: 

![image](https://user-images.githubusercontent.com/19702770/235527742-aa5dfe8d-997a-4bca-852f-6872440a0473.png)

When I rename the file name (in the folder) from "TML-12693-2-20221229-062004.mp4" to "20221229-062004.mp4" it starts to play without problems:

![image](https://user-images.githubusercontent.com/19702770/235527833-2179d244-72fa-42cc-bab5-75743e969080.png)

Is this a bug? any fix?

Regards,

Miguel

**Comment by yashtandon113** [comment, marked as answer]
Hi @MP3GMike
Thank you for bringing this up.

You can use the following name: TML_12693_2_20221229-062004.mp4.

I'm not sure what the reason is. It appears to be related to a special character, but I will discuss with my colleague first.
